### ISSUE TITLE
Install the Appsody operator using the Kabanero operator install yamls / scripts

### ISSUE THREAD

**Issue by kaczyns** [kabanero-io/kabanero-operator#94]
The Appsody operator should be added to the Kabanero operator install steps, as the Appsody operator will be used by the collections when deploying applications.

Instructions for the `daily` releases are here:
https://github.com/appsody/appsody-operator/tree/master/deploy/releases/daily
When a named release is cut we can pull those yamls and add to our installation instructions.

**Comment by kaczyns** [comment]
The following can be used to get started installing the Appsody operator:
https://github.com/appsody/appsody-operator/tree/master/deploy/releases/daily#operator-installation

Our dependencies yaml should be updated to include these things.  As I understand it, we have a script that generates our dependency script, so that will need to change as well.

Hopefully soon there will be an actual named release of the operator, at which point we can deliver our changes to the dependencies yaml (I am not comfortable delivering while it's pointing at a daily release).

**Comment by ianpartridge** [comment]
@seabaylea and I met with @arthurdm this morning to discuss the Appsody operator and what ships as part of the collection versus the operator itself.

I want to make sure we're all on the same page here - can @seabaylea and @arthurdm confirm they're happy with @kaczyns 's plan above ^^^^^^^^^ ?

**Comment by arthurdm** [comment]
ya - if we're referring to the Appsody Operator installation YAMLs (instead of making copies) then that's a good approach.  It allows us to do something similar to an ifix (i.e. patching a release) if we find bugs after the release is out.

**Comment by kyleschlosser** [comment]
The Kabanero operator needs to embed any resources that it uses, so we need to coordinate releases somewhat.

**Comment by marikaj123** [comment]
According to the Operator team, the expected Appsody operator release info was delivered on time and the status remains green.

**Comment by kaczyns** [comment]
This is the version of the Appsody operator that the Kabanero operator will be pulling in:
https://github.com/appsody/appsody-operator/tree/master/deploy/releases/0.1.0#operator-installation

**Comment by kaczyns** [comment]
The appsody operator instructions contain this statement:
```
$ OPERATOR_NAMESPACE=`<SPECIFY_OPERATOR_NAMESPACE_HERE>`
$ WATCH_NAMESPACE=`<SPECIFY_WATCH_NAMESPACE_HERE>`
```
When we copy this yaml over into the Kabanero operator, we'll want to replace these lines with `Meta.namespace` (I might have that name wrong, but there's some way to tell it that we should take the name of the namespace this is deployed in, and put it here).  The operator will be deployed to the namespace where the `Kabanero` instance is deployed, and will also be watching the same namespace.

**Comment by navidsh** [comment]
I am not sure if setting both `OPERATOR_NAMESPACE` and `WATCH_NAMESPACE` to `kabanero` instance's namespace is what we want to promote as a best practice. Since that means Jane would need to have access to the Kabanero namespace to deploy `AppsodyApplicaition` CR.

Might be better to set `WATCH_NAMESPACE="''"` so that the appsody operator watches all namespaces. This way Jane can deploy a CR in her teams namespace?

/cc @arthurdm

**Comment by arthurdm** [comment]
I agree with @navidsh - we will definitely want to deploy applications outside of the `kabanero` namespace.  

Can you expose the options (namespace to install and namespace to watch | watchAll) to kabanero users, so they get to pick?

**Comment by kaczyns** [comment]
Yes, we can, but for this initial release my understanding was that this all runs in a single namespace.

**Comment by kaczyns** [comment]
The Appsody operator is deployed with #118.
